Linear Search Algorithm/linear_search.js

```diff
@@ -0,0 +1,18 @@
+// Not using node.js, so, it is not possible for viewing output in terminal. Please view the output in the browser console.
+
+// We create an array of numbers
+const arrayToBeSearched = [3, 4, 7, 5, 11, 6, 13];
+//we are not looping and taking in inputs from the user as we are not using node, because of which input from terminal can't be taken.
+
+// We create a function for the Linear Search Algorithm
+const linearSearch = (array, num) => {
+  for (let i = 0; i < array.length; i++) {
+    if (array[i] === num) {
+      return i;
+    }
+  }
+  return -1;
+};
+
+linearSearch(numbers, 7); //we call the function with the array and the number we want to search (7 in this case). It will return the index, ie, 2.
```

Comment:
numbers variable is not declared

Reply:
Oh my bad. I changed the variable name at the last moment but forgot to change it while calling the function. Let me do it now.